package.json

```diff
@@ -71,5 +75,21 @@
         "protobufjs": "^7.2.4",
         "semver": "^7.5.3",
         "nx": "16.5.5"
-    }
+    },
+    "lint-staged": {
+        "packages/**/src/**/*.ts": [
+          "prettier --write"
+        ],
+        "!(packages/**/src/dist/)*.ts": [
+            "prettier --write"
+        ],
+        "!(packages/**/src/node_modules/)*.ts": [
```

Comment:
Same as above, is this just a nice to have for local troubleshooting view?

Reply:
Yes, i believe so.